classes/AddressFormat.php

```diff
@@ -49,13 +52,16 @@ class AddressFormatCore extends ObjectModel
         ),
     );
 
+    /** @var array Default required form fields list */
     public static $requireFormFieldsList = array(
```

Comment:
Should be ``$requiredFormFieldList`` but this is a breaking change ...  ping @julienbourdeau